syntax/fish.vim

```diff
@@ -2,6 +2,9 @@ if exists('b:current_syntax')
     finish
 endif
 
+syntax sync clear
+syntax sync fromstart
```

Comment:
Is this necessary? It can be slow for large files, or Vim would just do it by default for all filetypes. It should only really be used for things like heredocs, which last I checked fish didn't have, but I guess it has multiline strings that can be used similarly. But how often do you have strings span more than a few lines? Do you really need to sync all the way from start?